cmake/setup_write_config.cmake

```diff
@@ -225,6 +225,10 @@ FOREACH(_feature ${_deal_ii_features_sorted})
       _detailed("#            TBB_VERSION = ${TBB_VERSION}\n")
     ENDIF()
 
+    IF(_feature MATCHES "KOKKOS")
```

Comment:
is this also printing info if kokkos is disabled?

Reply:
No, it doesn't (but we agreed on making `Kokkos` a required dependency anyway).

Reply:
oh, right.